scripts/uploadIncrDB.py

```diff
@@ -0,0 +1,392 @@
+#!/usr/bin/env python3
+# Copyright (C) 2019 Zilliqa
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import mmap
+import time
+import re
+import os
+import shutil, sys
+import subprocess
+import tarfile
+import json
+import datetime
+from pathlib import Path
+import socket
+import argparse
+from urllib import request, parse
+
+PERSISTENCE_SNAPSHOT_NAME='incremental'
+STATEDELTA_DIFF_NAME='statedelta'
+BUCKET_NAME='BUCKET_NAME'
+NUM_TXBLOCK = 1
+NUM_DSBLOCK= "PUT_INCRDB_DSNUMS_WITH_STATEDELTAS_HERE"
+NUM_FINAL_BLOCK_PER_POW= "PUT_NUM_FINAL_BLOCK_PER_POW_HERE"
+SOURCE = './'
+TESTNET_NAME= "TEST_NET_NAME"
+SYNC_INTERVAL = 1
```

Comment:
Can we convert some of the variables here (e.g., `BUCKET_NAME`, `TESTNET_NAME`) into command-line options? Unlike the real constants that are not subject to change, they are actually variables that need meaningful values before running.